### DISCUSSION TITLE
Size prop not working on icon

### DISCUSSION THREAD

**Discussion by PoojMenon** [twilio-labs/paste#1311] [Q&A] [answered]
Hi, 

I've been trying to use the newly added credit icon, but for whatever reason, the size prop doesn't seem to be working with it. 

The icon itself doesn't change size, rather the container does. 

Link to [Codesandbox](https://codesandbox.io/s/dreamy-darwin-f2vtn?file=/pages/icon-test.js) where I've compared the size props.

Screenshot of icon comparisons:
![image](https://user-images.githubusercontent.com/72031986/112294748-11afaa80-8c8b-11eb-9d3d-84472eccbb61.png)

Am I using it wrong?

**Comment by TheSisb** [comment]
Hi @PoojMenon,

~This appears to be a bug with the framework used in the prototyping kit (NextJS).  NextJS uses SVGO to optimize inlined SVGs which removes the `viewBox` attribute by default (https://github.com/svg/svgo/issues/505) and causes the scaling to break.  Rest assured that the scaling should work outside of the prototyping kit. I've made a ticket to fix this in the future. Sorry for the inconvenience.~

Nevermind! I realized that the DeleteIcon had the `viewBox` intact and saw that the `viewBox` was simply missing in the source creditcard icon.  I've updated the ticket to fix the CreditCard icon and our guidance for icon addition: 
https://codesandbox.io/s/thirsty-cache-hv2s3

**Reply by PoojMenon** [reply]
Thanks Shadi - is there a timeline for the fix? We're currently in development and need to use this icon. Is there something we could do as a workaround in the meanwhile?

**Reply by TheSisb** [reply]
Hi @PoojMenon,

We don't have a timeline yet. When do you need the fix by? We can work around your schedule.

**Comment by serifluous** [comment, marked as answer]
@PoojMenon We'll actually get the fix in for this in PR #1334
